src/pages/sso/sso.ts

```diff
@@ -10,10 +10,13 @@ import { ToastController } from 'ionic-angular';
 export class SSOPage {
     pageTitle: string;
 
-    constructor(public toastCtrl: ToastController, public navCtrl: NavController, public navParams: NavParams) {
+    constructor(public navCtrl: NavController, public navParams: NavParams) {
         this.navCtrl = navCtrl;
         this.pageTitle = navParams.get('linkParam');
     }
 
-    ionViewDidEnter(): void {}
+    ionViewWillEnter(): void { 
+        window.open('https://docs.aerogear.org/aerogear/latest/keycloak/index.html?sso=1', '_system')
```

Comment:
so this will open the webpage in the system browser? In other platforms this page is in the app's webview?

Reply:
I was just chatting to @StephenCoady going to update it now to open in app

Reply:
@ciaranRoche np. Thanks.